src/configurefromcluster.ts

```diff
@@ -3,10 +3,10 @@
 import { TextDocumentContentProvider, Uri, EventEmitter, Event, ProviderResult, CancellationToken } from 'vscode';
 import { Shell } from './shell';
 import { FS } from './fs';
-import { Advanceable, Errorable, UIRequest, StageData, OperationState, OperationMap, advanceUri as wizardAdvanceUri, selectionChangedScript as wizardSelectionChangedScript, script, waitScript, styles } from './wizard';
+import { Advanceable, Errorable, UIRequest, StageData, OperationState, OperationMap, advanceUri as wizardAdvanceUri, selectionChangedScript as wizardSelectionChangedScript, script, waitScript, styles, extend } from './wizard';
 import { Context, getSubscriptionList, setSubscriptionAsync } from './azure';
 
-export const uriScheme : string = "acsconfigure";
+export const uriScheme : string = "k8sconfigure";
```

Comment:
somewhere the string constant is surrounded by single quote, and otherwhere double quote.  And also there are other code style consistent problems. Can we use travis ci to enable tslint validation before mergeing PR? If you're not familar with travis ci, i would like to provide a PR for this.

Reply:
A PR for CI would be great!  Thank you!  We'd also want to add this rule to the tslint config.